enterprise/frontend/src/embedding-sdk/store/auth.ts

```diff
@@ -16,23 +16,31 @@ import { getFetchRefreshTokenFn } from "./selectors";
 export const initAuth = createAsyncThunk(
   "sdk/token/INIT_AUTH",
   async (sdkConfig: SDKConfig, { dispatch }) => {
+    if ("jwtProviderUri" in sdkConfig) {
```

Comment:
@npretto let me know if this is what you have in mind for letting devs know about the change

Reply:
<img width="1430" alt="image" src="https://github.com/user-attachments/assets/abb4d6f9-ebb1-43e0-87b5-2acf97e6eca7">

Ouch, I thought so but the sdk usage problem is taking a lot more space than this one, we should probably add it to the usage problems here so that the user doesn't have to read all of the errors before reading the actionable one: https://github.com/metabase/metabase/blob/2c12222fc401781847886d30ab6c67a1f84fdcbd/enterprise/frontend/src/embedding-sdk/lib/usage-problem.ts#L33

Reply:
oh that's infinitely better. thanks for the callout

Reply:
alright, let me know if that looks better to you